TESTING.md

```diff
@@ -14,33 +14,81 @@ Note that because the logs are from GNOME Shell, there will be messages from all
+- [ ] `Super` `G` floats and then re-tiles a window
+- [ ] `Super` `Q` Closes a window
+- [ ] `Super` `M` Maximizes and un-maximizes a window
+- [ ] Turn off auto-tiling. New windows launch floating.
+- [ ] Turn on auto-tiling. Windows automatically tile.
+- [ ] Disabling and enabling auto-tiling correctly handles minimized, maximized, fullscreen, floating, and non-floating windows
 
 ### Workspaces
 
-- [ ] Tiled windows move across workspaces
+- [ ] Windows can be moved to another workspace with the keyboard
+- [ ] Windows can be moved to another workspace with the mouse
+- [ ] Windows can be moved to workspaces between existing workspaces
+- [ ] Moving windows to another workspace re-tiled the previous and new workspace
+- [ ] Active hint is present on the new workspace and once the window is returned to its previous workspace
 - [ ] Floating windows move across workspaces
-- [ ] Windows can be moved with the mouse and keyboard
-- [ ] Windows correctly move workspaces when a workspace is destroyed
+- [ ] Remove windows from the 2nd worspace in a 3 workspace setup. The 3rd workspace becomes the 2nd workspace, and tiling is unaffected by the move.
 
 ### Displays
 
-- [ ] Unplugging and plugging in displays re-tiles all workspaces correctly
-- [ ] Primary display changes are working correctly
-- [ ] Windows correctly move across workspaces after display changes
-
-### Tiling
-
-- [ ] Windows can be rearranged with the mouse and keyboard
-- [ ] Windows can be swapped with the mouse and keyboard
-- [ ] Try all forms of resizing a fork with the mouse and keyboard
-- [ ] Minimizing a window should detach it from the tree
-- [ ] Fullscreening, unfullscreening, maximizing, and unmaximizing are all working
-- [ ] Restarting the extension; or toggling auto-tile; correctly handles minimized, maximized, fullscreen, floating, and non-floating windows
+- [ ] Windows move across displays in adjustment mode with direction keys
+- [ ] Windows move across displays with the mouse
+- [ ] Changing the primary display moves the top bar. Window heights adjust on all monitors for the new position.
+- [ ] Unplug a display - windows from the display retile on a new workspace on the remaining display
+- [ ] Plug in a display - windows and workspaces don't change
+- [ ] NOTE: Add vertical monitor layout test
 
 ### Launcher
 
-- [ ] Windows should appear on launch
+- [ ] All windows on all workspaces appear on launch
+- [ ] Choosing an app on another workspace moves workspaces and focus to that app
+- [ ] Launching an application works
 - [ ] Typing text and then removing it will re-show those windows
 - [ ] Search works for applications and windows
 - [ ] The overlay hint correctly highlights the selected window
+- [ ] t: executes a command in a terminal
+- [ ] : executes a command in sh
+- [ ] = calculates an equation
+
+### Window Titles
+
+- [ ] Disabling window titles in Firefox works (Check debian and flatpak packages)
+
+## With Tiling Disabled
```

Comment:
Perhaps adding a check that half tiling still works with tiling disabled would be a good test as well.

Reply:
This is a living document that will constantly change. Approve the PR and add test cases as deemed necessary.